invisible_cities/types/ic_types_test.py

```diff
@@ -47,6 +47,31 @@ def test_minmax_does_not_accept_min_greater_than_max(a,b):
         with raises(AssertionError):
             minmax(a,b)
 
+@given(sensible_floats, sensible_floats)
+def test_minmax_getitem(lo, hi):
+    lo, hi = sorted([lo, hi])
+    mm = minmax(lo, hi)
+    assert lo == mm[0] == mm.min
+    assert hi == mm[1] == mm.max
+
+
+@given(minmaxes)
+def test_minmax_bracket(mm):
+    bracket = mm.max - mm.min
+    assert mm.bracket == bracket
+
+
+@given(minmaxes)
+def test_minmax_center(mm):
+    center = (mm.max + mm.min)*0.5
+    assert mm.center == center
+
+
+@given(minmaxes)
+def test_minmax_eq(mm):
+    assert mm == mm
```

Comment:
Eh? `assert mm == mm`? You are testing that any minmax equals itself. OK, but that's a far cry from actually testing the correct behaviour of `==`!

Reply:
Yes, that's quite embarrassing... Actual test coming up.